docs/examples/apt-cacher-ng.md

```diff
@@ -30,6 +30,8 @@ Use the following Dockerfile:
     # and then you can run containers with:
     #   docker run -t -i --rm -e http_proxy http://dockerhost:3142/ debian bash
     #
+    # Here, `dockerhost` is the IP address or FQDN of a host running docker daemon
```

Comment:
@moxiegirl @thaJeztah thinking out loud here, but do we have support for abbreviation on the documentation ? (`abbr` : https://developer.mozilla.org/en-US/docs/Web/HTML/Element/abbr)